aksel.nav.no/website/pages/templates/soknad-oppsummeringsside-real-examples/aap.tsx

```diff
@@ -92,7 +92,7 @@ function Example() {
                 <FormSummary.Value>Luke Skywalker</FormSummary.Value>
               </FormSummary.Answer>
               <FormSummary.Answer>
-                <FormSummary.Label>Personnummer</FormSummary.Label>
+                <FormSummary.Label>Fødselsnummer</FormSummary.Label>
```

Comment:
Not sure if we should change it here as well? This is probably copy-paste from the actual AAP application?

Reply:
Think it makes sense to just update everything 📝

Reply:
@sjur-gr What do you think?